src/app/app.module.ts

```diff
@@ -17,6 +17,11 @@ import { FlashcardComponent } from './pages/flashcard/flashcard.component';
 import { MessagesComponent } from './pages/messages/messages.component';
 import { AddCardComponent } from './pages/add-card/add-card.component';
 import { ForumsComponent } from './pages/forums/forums.component';
+import { CameraViewComponent } from './pages/camera-view/camera-view.component';
+import { ContinuePageComponent } from './pages/continue-page/continue-page.component';
+import { FinalConfirmComponent } from './pages/final-confirm/final-confirm.component';
+import { SettingsComponent } from './pages/settings/settings.component';
```

Comment:
Actually, would UserSettings be more descriptive?  Not sure if they'd ever be able to toggle any other than that (e.g. privacy settings, card settings, etc etc... or would that all fall under the umbrella "settings" component?).

Reply:
Completed as per pull request